hoomd/mpcd/module.cc

```diff
@@ -24,6 +24,9 @@
 // integration
 #include "Integrator.h"
 
+// ReverseNonequilibriumShearFlow
+#include "ReverseNonequilibriumShearFlow.h"
```

Comment:
I have changed the way `module.cc` is configured, so we will need to make some updates before opening a PR onto `glotzerlab/hoomd-blue`. But, no need to take action now.